WORKSPACE

```diff
@@ -28,11 +31,3 @@ load(
 )
 
 apple_support_dependencies()
-
```

Comment:
Why is this no longer needed?

Reply:
This is included by rules_apple by default now